File: arkts_static_test/03_Types/ets_cases/3.17_Tuple_Types/cross_lang_verify/runtime/TYP_03_17_013.java
/**
 * Java cross-language verification for TYP_03_17_013_RUNTIME_TUPLE_AS_TUPLE
 * Java 没有 Tuple 超类型
 */
class TYP_03_17_013 {
    public static void main(String[] args) {
        Object[] tuple = {1, "hello"};
        Object t = tuple;

        System.out.println("verified");
    }
}

File: arkts_static_test/03_Types/ets_cases/3.17_Tuple_Types/cross_lang_verify/runtime/TYP_03_17_013.swift
/**
 * Swift cross-language verification for TYP_03_17_013_RUNTIME_TUPLE_AS_TUPLE
 * Swift 没有 Tuple 超类型
 */
import Foundation

let tuple: (Int, String) = (1, "hello")
let t: Any = tuple

print("verified")
